apps/wing/src/commands/pack.test.ts

```diff
@@ -186,10 +186,18 @@ describe("wing pack", () => {
     await extractTarball(join(outdir, tarballPath), outdir);
 
     // symlink node_modules/@winglang/sdk to our version of the sdk so the import works
-    await fs.mkdir(join(outdir, "package", "node_modules", "@winglang"), { recursive: true });
+    await fs.mkdir(join(outdir, "package", "node_modules", "@winglang", "sdk", "lib"), {
+      recursive: true,
+    });
+
+    await fs.symlink(
+      require.resolve("@winglang/sdk/lib/index.js"),
+      join(outdir, "package", "node_modules", "@winglang", "sdk", "index.js")
+    );
+
     await fs.symlink(
-      require.resolve("@winglang/sdk"),
-      join(outdir, "package", "node_modules", "@winglang", "sdk")
+      require.resolve("@winglang/sdk/lib/macros.js"),
+      join(outdir, "package", "node_modules", "@winglang", "sdk", "lib", "macros.js")
```

Comment:
See my other comment -- if `src/macro.ts` is compiled with the rest of the files (and exported in `src/index.ts`) then maybe we won't need to special case it here